audio-processing/wav2vec2-fine-tuning-checkpoint.ipynb

```diff
@@ -46,17 +46,9 @@
   {
```

Comment:
<p>Same comment, remove: "A Poplar SDK environment enabled (see the&nbsp;<a href="https://docs.graphcore.ai/en/latest/getting-started.html" target="_blank">Getting Started</a>&nbsp;guide for your IPU system)"</p><p> </p><p>And move Running on Paperspace section to where it is.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>